File: CloudFlow/FrontEnd-App/src/app/shared/model/bpm/notifications/hierarchical-superior-of-user-metadata-transition-notification.model.ts
import { TransitionNotificationModelType } from "./transition-notification-model-type";
import { TransitionNotificationModel } from "./transition-notification.model";
import { JsonObject, JsonProperty } from "json2typescript";

@JsonObject
export class HierarchicalSuperiorOfUserMetadataTransitionNotificationModel extends TransitionNotificationModel {

	public constructor() {
		super(TransitionNotificationModelType.HIERARCHICAL_SUPERIOR_OF_USER_METADATA);
	}

	@JsonProperty("metadataName", String)
	public metadataName: string;
}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/bpm/notifications/HierarchicalSuperiorOfUserMetadataTransitionNotificationModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.bpm.notifications;

public class HierarchicalSuperiorOfUserMetadataTransitionNotificationModel extends TransitionNotificationModel {
	
	private static final long serialVersionUID = 1L;

	private String metadataName;
	
	public HierarchicalSuperiorOfUserMetadataTransitionNotificationModel() {
		super(TransitionNotificationModelType.HIERARCHICAL_SUPERIOR_OF_USER_METADATA);
	}

	public String getMetadataName() {
		return metadataName;
	}

	public void setMetadataName(String metadataName) {
		this.metadataName = metadataName;
	}
}
